modules/controlfreec/1.1/controlfreec.smk

```diff
@@ -160,89 +222,128 @@ rule _controlfreec_config:
         "sed \"s|bedtoolsPath|$bedtoolsPathName|g\" | "
         "sed \"s|samtoolsPath|$samtoolsPathName|g\" | "
         "sed \"s|breakPointValue|{params.breakPointValue}|g\" | "
+        "sed \"s|breakPointTypeBe|{params.breakPointType}|g\" | "
         "sed \"s|coefVar|{params.coefVar}|g\" | "
         "sed \"s|numCon|{params.numCon}|g\" | "
         "sed \"s|booCon|{params.booCon}|g\" | "
         "sed \"s|forceGCvalue|{params.forceGC}|g\" | "
+        "sed \"s|interPoly|{params.intercept}|g\" | "
         "sed \"s|numDegree|{params.degree}|g\" | "
         "sed \"s|minCNAvalue|{params.minCNAlength}|g\" | "
+        "sed \"s|minMapPerWindow|{params.minMapPerWindow}|g\" | "
         "sed \"s|minimumSubclonePresenceValue|{params.minimumSubclonePresence}|g\" | "
+        "sed \"s|minCovPerPos|{params.minimalCoveragePerPosition}|g\" | "
         "sed \"s|booNoise|{params.noisyData}|g\" | "
         "sed \"s|stepValue|{params.step}|g\" | "
         "sed \"s|teloValue|{params.telocentromeric}|g\" | "
+        "sed \"s|uniqBoo|{params.uniqBoo}|g\" | "
         "sed \"s|numThreads|{params.threads}|g\" | "
         "sed \"s|referenceFile|{input.reference}|g\" > {output}"
 
 
 rule _controlfreec_run:
```

Comment:
I'm unsure how this is going to work in unpaired mode, e.g. without a normal. I think you might need a separate rule.

Reply:
Yeah I think the simplest method would be to have a bunch of different config_WGS.txt files for different contexts (ex. paired, unpaired, WES, WGS). And a separate rule for unpaired vs paired to account for the control.

Reply:
At the moment, I made control-freec run in paired mode always and to match unpaired tumours since it gives the cleanest results.